JSDemos/Demos/Charts/BiDirectionalBarChart/description.md

```diff
@@ -1 +1,11 @@
-This demo shows how to create a bi-directional bar chart. This is a rotated [stacked bar chart](/Demos/WidgetsGallery/Demo/Charts/StackedBar/jQuery/Light/) with half of its data source values converted from positive to negative.
\ No newline at end of file
+To create a bi-directional bar chart, follow the steps below:
+
+1. Convert half of the data source values from positive to negative. In this demo, the `male` percentages are negative, while the `female` are positive.
+
+2. Implement a rotated [stacked bar chart](/Demos/WidgetsGallery/Demo/Charts/StackedBar/jQuery/Light/). Use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object to specify [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#type) and the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#argumentField) property (`age` in this case). Set the [rotated](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#rotated) property to `true`.
+
+3. Declare two objects in the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) array: one for negative values, the other for positive. Specify [color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#color), [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#name) and a [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField) for each series.
+
+4. Change the [label](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/label/) text of the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/) so the negative values appear as positive. To implement this technique, use the [customizeText](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/label/#customizeText) function.
+
+To further customize your bi-directional bar chart, implement the [tooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/) and change the [legend](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/).
```

Comment:
1. Are you only implementing one tooltip ("implement the tooltip")? Or do you mean "implement tooltips"? Just checking.

2. Regarding the legend, can we make it more specific than just "change the legend"? Can we specify what can be changed in the legend?

Reply:
I changed this part, could you pleas have a look?

Reply:
Thanks for updating the text. Sounds good to me. :)